src/main/java/nextstep/auth/AuthService.java

```diff
@@ -0,0 +1,30 @@
+package nextstep.auth;
+
+import nextstep.support.MemberNotFoundException;
+import nextstep.member.MemberDao;
+import nextstep.support.UnauthorizedException;
+import org.springframework.stereotype.Service;
+
+@Service
+public class AuthService {
+
+    private final static JwtTokenProvider TOKEN_PROVIDER = new JwtTokenProvider();
+    private MemberDao memberDao;
```

Comment:
변경되지 않는 객체에는 final을 선언해주시면 좋을 것 같아요 :)
```suggestion
    private final MemberDao memberDao;
```